Program.cs

```diff
@@ -1,4 +1,16 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
+Console.Write ("Enter a word or phrase:");
+string text = Console.ReadLine ();
+text = text.Replace (" ", "").ToLower ();
+char[] ch = text.ToCharArray ();
+Array.Reverse (ch);
+string rev = new string (ch);
+bool b = text.Equals (rev); 

```

Comment:
You can call it directly in the If condition because the 'b' variable is used it here only.